util/etcd/etcd.go

```diff
@@ -333,3 +334,10 @@ func keyWithPrefix(prefix, key string) string {
 
 	return path.Join(prefix, key)
 }
+
+// SetEtcdCliByNamespace is used to add an etcd namespace prefix before etcd path.
+func SetEtcdCliByNamespace(cli *clientv3.Client, namespacePrefix string) {
```

Comment:
Some package may use etcd client directly, rather that depending on this util/etcd.

In that case, the code need to be updated case by case.

Reply:
This pr introduces keyspace and basic utils, and other components will be modified in the specified PR.